gp-copy-cat/gpcc-copy-product-value.js

```diff
@@ -0,0 +1,22 @@
+/**
+* Gravity Perks // GP Copy Cat // Copy Product Value Without the Price
+* https://gravitywiz.com/documentation/gravity-forms-copy-cat/
+*
+* Copy the value of a choice based product field without the price.
+*
+* Instructions:
+* 1. Add an HTML field to your form.
+* 2. Copy and paste the entire content of this snippet into the "Content" field setting.
+* 3. Update the target field ID within the snippet.
```

Comment:
Can we update these instructions to use our [Custom Javascript](https://gravitywiz.com/gravity-forms-custom-javascript/) plugin instead?